books/RayTracingInOneWeekend.html

```diff
@@ -4126,7 +4125,7 @@
 
 
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ C++ highlight
-        cam.defocus_angle = 10;
+        cam.defocus_angle = 10.0;
```

Comment:
We generally (but not always) omit decimal points for integer values, and where C++ type resolution can implicitly handle the conversion at compile time.

Reply:
For example, `color(0,0,0)`, `color(1,1,1)`, and so forth.

One exception would be when using the `auto` keyword, where the scalar type is required.

Reply:
Hah! Later on:
```
        cam.focus_dist    = 10.0;
```

Oy vey. I'm so bad.

Reply:
Now I'm for keeping your change.